accounts-db/src/accounts_db.rs

```diff
@@ -15847,6 +15836,7 @@ pub mod tests {
                 (&0, 2, 1280)
             );
         }
+        accounts.accounts_index.set_startup(Startup::Normal);
```

Comment:
Looks like `cargo fmt` didn't like this line. 
Is this line relevant to this pr change?

Reply:
Yes, required. I fixed it already.